corehq/apps/export/custom_export_helpers.py

```diff
@@ -234,8 +234,11 @@ def is_special_type(q):
 
         def generate_additional_columns(requires_case):
             ret = []
+            # the default of the case_name_col MUST have selected = false, because
+            # there is no way to store selected = false in the current export schema
+            # as it only stores selected columns, not the selected states for all columns
```

Comment:
1) all current exports - not unless the export is resaved, this is not changing any existing export schemas
2) for new exports - the default behavior will be that case_name is unchecked. once the user checks it, then it will be added to the export, but it will not initially by default be checked.
3) for existing exports - If the user created the export with case_name unchecked previously and never reloaded the modify form, then the correct behavior will now occur. If the form was saved with case_name checked previously, then it will remain checked and be included in the export.

Again, what I touched did not have anything to do with the actual processing / rendering of exports. It simply changed how the user interacts create/modify export form, how it is initialized, and how it sends data back to the server to be used export schema.